### PR TITLE
gthumb - update to version 3.12.4

### PR DESCRIPTION
**Pull request by fritzkink**

### CODE REVIEW

**Comment by AndWac** [comment]
Please check your PR. If I run gmake sample-manifest I'll get some more files and when I update the manifest and run gmake REQUIRED_PACKAGES I get additional dependencies:
diff --git a/components/image/gthumb/Makefile b/components/image/gthumb/Makefile
index 20d4d79089..aaa0baa575 100644
--- a/components/image/gthumb/Makefile
+++ b/components/image/gthumb/Makefile
@@ -69,8 +69,11 @@ REQUIRED_PACKAGES += library/desktop/cairo
 REQUIRED_PACKAGES += library/desktop/gdk-pixbuf
 REQUIRED_PACKAGES += library/desktop/gtk3
 REQUIRED_PACKAGES += library/desktop/pango
+REQUIRED_PACKAGES += library/desktop/webkitgtk2
 REQUIRED_PACKAGES += library/glib2
 REQUIRED_PACKAGES += library/lcms2
+REQUIRED_PACKAGES += library/libsecret
+REQUIRED_PACKAGES += library/libsoup
 REQUIRED_PACKAGES += library/libwebp
 REQUIRED_PACKAGES += library/zlib
 REQUIRED_PACKAGES += system/library

File: components/image/gthumb/pkg5 [hunk 1]
```diff
@@ -21,7 +22,8 @@
         "system/library/g++-13-runtime",
         "system/library/gcc-13-runtime",
         "system/library/math",
-        "text/itstool"
+        "text/itstool",
+        "x11/library/libx11"
     ],
     "fmris": [
         "image/viewer/gthumb"
```
Comment (AndWac) [inline components/image/gthumb/pkg5:29]: pkg5 update is missing your latest changes

**Comment by mtelka** [comment]
Ah, non-linear git history:
```
* commit c5aab181151d729f313e9df78fae70b9dce43528
| Author: hoewweken <44616757+hoewweken@users.noreply.github.com>
| Date:   Mon Nov 27 10:14:49 2023 +0100
| 
|     php-imagick add ini file
|   
*   commit 538d58af513e8ebeee2105dfcd2cd8969fa3ea23
|\  Merge: ddf8ab80eb 432be3990a
| | Author: Andreas Wacknitz <A.Wacknitz@gmx.de>
| | Date:   Mon Nov 27 07:42:44 2023 +0100
| | 
| |     Merge branch 'fritzkink-thumb' into oi/hipster
| | 
| * commit 432be3990a0f769fe21d3f72f563b2805cdefea1
|/| Merge: ddf8ab80eb aefb4c2971
| | Author: Andreas Wacknitz <A.Wacknitz@gmx.de>
| | Date:   Mon Nov 27 07:41:12 2023 +0100
| | 
| |     Merge branch 'thumb' of https://github.com/fritzkink/oi-userland into fritzkink-thumb
| | 
| * commit aefb4c29714c7409f8fa9d4f2c5063c922aec561
| | Author: fritzkink <friedrich.kink@fkink.de>
| | Date:   Sun Nov 26 11:10:32 2023 +0100
| | 
| |     gthumb - update to version 3.12.4
| | 
* | commit ddf8ab80eb326ece51518e0a3d148c6237160b91
| | Author: Marcel Telka <marcel@telka.sk>
| | Date:   Sun Nov 26 23:21:41 2023 +0100
| | 
| |     build-essential: add back openjdk8
```
@AndWac, I hope this is just a misclick and nothing that will appear regularly :-).  Thank you.

**Comment by AndWac** [comment]
Am 27.11.23 um 14:58 schrieb Marcel Telka:
>
> Ah, non-linear git history:
>
> |* commit c5aab181151d729f313e9df78fae70b9dce43528 | Author: hoewweken
> ***@***.***> | Date: Mon Nov 27
> 10:14:49 2023 +0100 | | php-imagick add ini file | * commit
> 538d58af513e8ebeee2105dfcd2cd8969fa3ea23 |\ Merge: ddf8ab80eb
> 432be3990a | | Author: Andreas Wacknitz ***@***.***> | | Date:
> Mon Nov 27 07:42:44 2023 +0100 | | | | Merge branch 'fritzkink-thumb'
> into oi/hipster | | | * commit
> 432be3990a0f769fe21d3f72f563b2805cdefea1 |/| Merge: ddf8ab80eb
> aefb4c2971 | | Author: Andreas Wacknitz ***@***.***> | | Date:
> Mon Nov 27 07:41:12 2023 +0100 | | | | Merge branch 'thumb' of
> https://github.com/fritzkink/oi-userland into fritzkink-thumb | | | *
> commit aefb4c29714c7409f8fa9d4f2c5063c922aec561 | | Author: fritzkink
> ***@***.***> | | Date: Sun Nov 26 11:10:32 2023 +0100 | |
> | | gthumb - update to version 3.12.4 | | * | commit
> ddf8ab80eb326ece51518e0a3d148c6237160b91 | | Author: Marcel Telka
> ***@***.***> | | Date: Sun Nov 26 23:21:41 2023 +0100 | | | |
> build-essential: add back openjdk8 |
>
> @AndWac <https://github.com/AndWac>, I hope this is just a misclick
> and nothing that will appear regularly :-). Thank you.
>
>
Sorry for that. It was my first try on changing a PR following the
Github instrcutions. I don't know why that went wrong.

**Pull request merged**